analysis/image_info.py

```diff
@@ -0,0 +1,247 @@
+from astropy.coordinates import SkyCoord
+from astropy.io import fits
+from astropy.time import Time
+from astropy.wcs import WCS
+
+# ImageInfo is a helper class tha wraps basic data extracted from a
+# fits image file.
```

Comment:
Since `fits` is an English word, expand the acronym here at the beginning of the file for clarity. This should also be all caps when in comments (this file uses a mix).

Reply:
Done.